wbia/dtool/sql_control.py

```diff
@@ -410,32 +412,37 @@ def __init__(self, ctrlr):
 
             @property
             def version(self):
-                return self.ctrlr.executeone(
-                    f'SELECT metadata_value FROM {METADATA_TABLE_NAME} WHERE metadata_key = ?',
-                    ('database_version',),
-                )[0]
+                stmt = f'SELECT metadata_value FROM {METADATA_TABLE_NAME} WHERE metadata_key = ?'
+                try:
+                    return self.ctrlr.executeone(stmt, ('database_version',))[0]
+                except IndexError:  # No result
+                    return None
 
             @version.setter
             def version(self, value):
                 if not value:
                     raise ValueError(value)
-                return self.ctrlr.executeone(
+                self.ctrlr.executeone(
                     f'INSERT OR REPLACE INTO {METADATA_TABLE_NAME} (metadata_key, metadata_value) VALUES (?, ?)',
                     ('database_version', value,),
                 )[0]
```

Comment:
Is `[0]` still necessary if you're not returning the version anymore?

Reply:
Good point!